reo/src/urdb_parse.py

```diff
@@ -639,7 +638,13 @@ def get_hours_in_month(self, month):
         return range(hours[month], hours[month + 1])
 
     def get_tou_steps(self, current_rate, month, period):
-
+        """
+        Get the TOU steps indexed on 1-8760 for hourly simulations
+        :param current_rate: RateData class instance
+        :param month: int
+        :param period: int
+        :return: list of ints
+        """
         step_array = []
         start_step = 1
         start_hour = 1
```

Comment:
delete also?

Reply:
is it wrong? (I was trying to document how the function is meant to be used.)

Reply:
oh haha, I thought the comment was going to show up more line specific. just the start_hour=1 part

Reply:
gotchya: removed it in https://github.com/NREL/REopt_Lite_API/pull/228/commits/f9e2ba7d5606a6777b329125ef04ae7d9b51f82b